applications/serial_lte_modem/doc/TCPIP_AT_commands.rst

```diff
@@ -1251,8 +1251,8 @@ Response syntax
    #XTCPCLI: <handle>,<data_mode>
 
 The ``<handle>`` value is an integer.
-When positive, it indicates that it opened successfully.
-When negative, it indicates that it failed to open.
+When positive,it indicates that it opened successfully.
+When negative,it indicates that it failed to open.
```

Comment:
These spaces should not be removed.

Reply:
Sorry, mistake due to quite a labor task